### PR TITLE
Fix automation column sorting

### PR DESCRIPTION
**Pull request by dhaiducek**
Unless someone has a better idea, it looks like we need to disable the Automation sorting since automation is queried in the child component and can't be extracted into table metadata like we do for other columns where we extract strings from the policy.

Storing table metadata in `utils.js`: https://github.com/open-cluster-management/grc-ui/blob/88845b1180e33d0dc080c5b8954c483d0eb53a17/src-web/tableDefinitions/utils.js#L136

Query in `AutomationButton`: https://github.com/open-cluster-management/grc-ui/blob/88845b1180e33d0dc080c5b8954c483d0eb53a17/src-web/components/common/AutomationButton.js#L32

Addresses https://github.com/open-cluster-management/backlog/issues/13794

### CODE REVIEW

**Comment by dhaiducek** [comment]
This isn't great since "Automation" would be the only column not sorted, but I haven't found a better solution yet.

**Review by willkutler — APPROVED** [review]

File: src-web/components/modals/AnsibleAutomationModal/AnsibleCredentialsSelection.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const renderAnsibleCredentialsSelection = ({
       {!ansCredentialFlag &&
         <div className='infoArea createCredential'>
           <Text>
-            {msgs.get('ansible.no.credential', locale)}
+            {`${msgs.get('ansible.no.credential', locale)} `}
```
Comment (ChunxiAlexLuo) [inline src-web/components/modals/AnsibleAutomationModal/AnsibleCredentialsSelection.js:64]: This is required exactly 8px by designer, I fixed this in another stylish pr by css so no need extra space here

**Review by ChunxiAlexLuo — CHANGES_REQUESTED** [review]
all rest looks good to me

File: src-web/components/modals/AnsibleAutomationModal/AnsibleCredentialsSelection.js (hunk 1 above)
Comment (dhaiducek) [reply src-web/components/modals/AnsibleAutomationModal/AnsibleCredentialsSelection.js]: Done

**Review by ChunxiAlexLuo — APPROVED** [review]
/lgtm

**Pull request merged**